gr-digital/lib/costas_loop_cc_impl.cc

```diff
@@ -31,9 +31,28 @@
 #include <gnuradio/sincos.h>
 #include <boost/format.hpp>
 
+#define CL_TWO_PI 6.28318530717958647692
```

Comment:
Guessing at least some of these are defined in <math.h> or <cmath> or whatever. Please use those instead of redefining here.

Reply:
michaelld: I double-checked.  Those particular defines are not in math.h.  The only Pi options there are:
# define M_PI		3.14159265358979323846	/* pi */
# define M_PI_2		1.57079632679489661923	/* pi/2 */
# define M_PI_4		0.78539816339744830962	/* pi/4 */
# define M_1_PI		0.31830988618379067154	/* 1/pi */
# define M_2_PI		0.63661977236758134308	/* 2/pi */
# define M_2_SQRTPI	1.12837916709551257390	/* 2/sqrt(pi) */

Not sure why 2*Pi wouldn't be a standard define.  You'd think that would be pretty common.

Reply:
#include <gnuradio/math.h>

GR_M_PI = <pi>
GR_M_TWOPI = (2*pi)

Reply:
@ghostop14 OK ... well, worth checking! At least those that @bistromath notes can be fixed!

Reply:
I'd suggest adding more constants to `<gnuradio/math.h>` as we see fit (since they might have some use in the future.

Reply:
Nvm, this all shows as outdated in GitHub, I shouldn't dig up corpses :)